src/documentation/pages/components/header/examples/_centurylink/_base_centurylink.vue

```diff
@@ -0,0 +1,82 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base_centurylink" padding="-p--0" :tabs="exampleTabs">
+    <Wrapper slot="example">
+      .-p--3(style='background-color:#eee;')
```

Comment:
I suggest using additionalClasses for this, like this:
<ComponentExample additionalClasses="-bg--grey-20"